src/Loader.php

```diff
@@ -39,8 +39,9 @@ public function init()
         parent::init();
 
         if (!$this->shim) {
-            $this->shim = ['View', 'class' => ['padded segment'], 'style' => ['min-height' => '7em']];
+            $this->shim = ['View', 'class' => ['ui active centered inline loader']];
```

Comment:
sometimes this shows double-loader. Also if we disable auto-load this shows loading indicator even though it's not loading.

Reply:
I'm ok leaving this for now like this.

Reply:
rolled back